javascripts/app.js

```diff
@@ -1,7 +1,7 @@
-$('#addPussy').click(function () {
-  const pussyImg = $('img').clone()[0];
-  /* Not sure what to do here :(
-   * Maybe something like:
-   * $('#catContainer').append();
-   */
+
+$(document).ready(function(){
```

Comment:
![img](http://cdn.thedailybeast.com/content/dailybeast/articles/2015/08/25/roger-ailes-trump-s-new-attack-on-megyn-kelly-is-unacceptable-and-disturbing/jcr:content/image.crop.800.500.jpg/48017699.cached.jpg)

You only had to change 1 line of code. But this will work.